articles/introduction-to-deep-learning/index.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/introduction-to-deep-learning/
+title: Introduction to Deep Learning
+description:
+author: judy-nduati
+date: 2020-09-16T00:00:00-13:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/introduction-to-deep-learning/hero.jpg
+    alt:
+---
+Deep Learning (DL) is becoming increasingly popular among IT enthusiasts due to its promising benefits. It is a method that enables image recognition, natural language processing(NLP), and voice recognition to take place. Have you ever wondered how machines function like the human brain? Well, that is thanks to the concept of Deep Learning. Deep Learning is a potent tool, and once you read this article, you will have a better understanding of what DL is and how it works.
+<!--more-->
+### Introduction
+### What is Deep Learning?
+
+![Venn diagram](/engineering-education/introduction-to-deep-learning/ai-ml-dl.jpg)<br>
+
+[*Image source*](https://towardsdatascience.com/cousins-of-artificial-intelligence-dda4edc27b55)
+
+Deep Learning is a subset of machine learning(ML), DL learns features and tasks directly from data such as images, text, or sound.
+
+To take it further, machine learning is a subset of artificial intelligence(AI). AI is the capability of machines and computers to mimic human intelligence and behavior.
+
+AI is accomplished by studying how the human brain operates while trying to solve a problem. ML is a branch of AI that gives machines and computers the ability to learn data without being programmed.
```

Comment:
Good point. Adding the fact that machine learning takes a statistical approach to obtain patterns from data would add additional weightage to the statement being made.